### ISSUE TITLE
Faster RCNN - Which approach is implemented? 

### ISSUE THREAD

**Issue by HutzelFutzel** [tensorflow/models#3894]
Hi,
which approach from the Faster RCNN Paper is implemented in the object detection? Alternating training, Approximate joint training or Non-approximate joint training?

**Comment by Abhijit-2592** [comment]
As far as I understand, none of the above. They have slightly tweaked the loss function so that the entire Architecture can be trained jointly. You can find more info on this in their official paper

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
